src/header/header.scss

```diff
@@ -245,15 +268,104 @@
       &:hover {
         svg.iui-header-button-icon {
           @include themed {
-            background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
+            background-color: rgba(
+              t(iui-color-foreground-primary-rgb),
+              t(iui-opacity-6)
+            );
           }
         }
       }
+
+      &:focus,
+      &:focus-visible {
+        @include themed {
+          box-shadow: t(iui-focus-box-shadow),
+            0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+        }
+      }
+
+      &,
+      &:focus:not(:focus-visible) {
+        @include themed {
+          box-shadow: 0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+        }
+      }
+    }
+  }
+
+  .iui-button-split-menu {
```

Comment:
It is not handling overflow like basic header button:
![image](https://user-images.githubusercontent.com/36186912/127850995-10083766-e4e3-4779-8568-ab4235438af5.png)